code/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -1066,6 +1062,18 @@
 
 	return .
 
+/obj/item/bodypart/proc/huskify_image(image/thing_to_husk, draw_blood = TRUE)
+	var/husk_color_mod = rgb(96, 88, 80)
```

Comment:
Can you make a define for this instead? colors.dm has a list of similair usecases